tests/integration_tests/security/test_spectre_meltdown.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# N.B.: Although this repository is released under the Apache-2.0, part of its test requires a
+# script from the third party "Spectre & Meltdown Checker" project. This script is under the
+# GPL-3.0-only license.
+"""Tests spectre / meltdown mitigation."""
+
+import pytest
+import requests
+
+from framework import utils
+from host_tools.network import SSHConnection
+
+
+CHECKER_URL = "https://meltdown.ovh"
+CHECKER_FILENAME = "spectre-meltdown-checker.sh"
+
+
+@pytest.fixture(name="spectre_meltdown_checker")
```

Comment:
I'm wondering if we could use ```scope="session"``` to avoid downloading it for every test.

Reply:
Since `tmp_path` fixture is function scope, it raises the following error.
```
ScopeMismatch: You tried to access the function scoped fixture tmp_path with a session scoped request object, involved factories:
integration_tests/security/test_spectre_meltdown.py:20:  def spectre_meltdown_checker(tmp_path)
../../usr/local/lib/python3.10/site-packages/_pytest/tmpdir.py:202:  def tmp_path(request: _pytest.fixtures.FixtureRequest, tmp_path_factory: _pytest.tmpdir.TempPathFactory) -> pathlib.Path
```

But to reduce test time, it's fair enough. I'll handle this without `tmp_path` fixture.

Reply:
I found `tmp_path_factory` is available for session-scope temporary directory.
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html#the-tmp-path-factory-fixture
Done.